### PR TITLE
Mobile: Fix cannot send docs, excel files and preview attachment

### PR DESCRIPTION
**Pull request by phatphanvan**

### CODE REVIEW

File: libs/store/src/lib/messages/messages.slice.ts [hunk 1]
```diff
@@ -460,22 +461,11 @@ export const sendMessage = createAsyncThunk('messages/sendMessage', async (paylo
...
-					const uploadPromises = createdFiles.map((file, index) => {
-						return handleUploadFile(client, session, clanId, channelId, file.name, file, index);
-					});
-
-					const uploadedNonDirectFiles = await Promise.all(uploadPromises);
-					uploadedFiles = [...uploadedFiles, ...uploadedNonDirectFiles];
+				if (isMobile) {
+					uploadedFiles = await getMobileUploadedAttachments({ attachments, channelId, clanId, client, session });
+				} else {
+					uploadedFiles = await getWebUploadedAttachments({ attachments, channelId, clanId, client, session });
```
Comment (phuocnguyenncc) [inline libs/store/src/lib/messages/messages.slice.ts:467]: re-recheck logic on DM

File: libs/utils/src/lib/utils/index.ts [hunk 2]
```diff
@@ -661,6 +665,56 @@ export async function fetchAndCreateFiles(fileData: ApiMessageAttachment[] | nul
...
+	attachments: ApiMessageAttachment[];
+	client: Client;
+	session: Session;
+	clanId: string;
+	channelId: string;
+}): Promise<ApiMessageAttachment[]> {
+	const { attachments, client, session, clanId, channelId } = payload;
+	if (!attachments || attachments?.length === 0) {
+		return [];
+	}
```
Comment (phuocnguyenncc) [inline libs/utils/src/lib/utils/index.ts:704]: const nonDirectAttachments = attachments.filter(
					(att) => !att.url?.includes(EMimeTypes.tenor) && !att.url?.includes(EMimeTypes.cdnmezon)
				);

Comment (phuocnguyenncc) [reply libs/utils/src/lib/utils/index.ts:704]: bổ sung

Comment (phatphanvan) [reply libs/utils/src/lib/utils/index.ts:704]: updated

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**